src/pke/include/schemebase/base-fhe.h

```diff
@@ -74,9 +74,10 @@ class FHEBase {
    * @param dim1 - vector of inner dimension in the baby-step giant-step routine
    * for encoding and decoding
    * @param slots - number of slots to be bootstrapped
+   * @param correctionFactor - value to rescale message by to improve precision. If set to 0, we use the default logic.
```

Comment:
we should say this is only used for NATIVE_SIZE=64 (not used for NATIVE_SIZE=128)